.github/workflows/ci.yml

```diff
@@ -102,9 +102,9 @@ jobs:
       - name: Run tests with coverage instrumentation
         run: |
             cargo llvm-cov clean --workspace
-            cargo llvm-cov --all-features --doctests
+            cargo llvm-cov --doctests
```

Comment:
This will bring the coverage down by a large margin.
It may be nice to spread
```rust
#![cfg_attr(coverage, feature(no_coverage))]
```
around.

(Note: That'll make `llvm-cov` run only on nightly, but that should be ok).